1 PR Accepted 1 PR Opened

wtrzas2 opened the pull request:
AxonFramework PR Accepted

AxonFramework PR Opened: `AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey`

Test failed before with commit `ce43ce465412d89b2f80b2bfbcd08fd186252f2a`
NonDex Log can be found on VM fa24-cs527-092.cs.illinois.edu within `ce43c-commit` branch: `/home/wtrzas2/AxonFramework/messaging/.nondex/5JK0fRotW3nz63ObYS9MJM3mDOwIK1sECutPeAWL0z0=/org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.txt`

Test passed after my 1st AnnotatedMessageHandlingMemberTest PR. 
NonDex Log can be found on VM fa24-cs527-092.cs.illinois.edu within `f933856-commit` branch:
`/home/wtrzas2/AxonFramework/messaging/.nondex/cEErZ8IzCaVUIdz2TOfTYZogm3ztcyugcj8Odj3lE+s=/org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.txt`

darko-marinov commented on pr-data.csv:
```diff
@@ -3472,9 +3472,9 @@ https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810ae
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a
-https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,,,
+https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,Opened,https://github.com/AxonFramework/AxonFramework/pull/3142,
```
3142 has been accepted. Did you run NonDex on the entire project to see if it finds some new tests?

wtrzas2 commented:
Yes that PR has been accepted, but it's the one I was referring to in the Canvas message I sent you and Ajitesh and you had mentioned to mark it as "Opened" so should I just switch it to "Accepted"? I just ran NonDex on the entire project and got this output:

[INFO] The following tests failed in the clean run:
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#publicationOrder
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#noMoreConsumersRegisteredWithUnitOfWorkWhenSecondEventIsPublished
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#publicationWithNestedUow
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#messageMonitorRecordsIngestionAndPublication_InUnitOfWork
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#consumersRegisteredWithUnitOfWorkWhenFirstEventIsPublished
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#commitOnUnitOfWork
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#publicationForbiddenDuringRootUowCommitPhase
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#publicationForbiddenDuringUowCommitPhase
[WARNING] org.axonframework.eventhandling.AbstractEventBusTest#dispatchInterceptor
[WARNING] org.axonframework.queryhandling.registration.LoggingDuplicateQueryHandlerResolverTest#throwsNoErrorOnDuplicateHandlerAndAddsItToList
[WARNING] org.axonframework.eventhandling.SimpleEventBusTest#eventIsDispatchedToSubscribedListeners
[WARNING] org.axonframework.eventhandling.scheduling.quartz.QuartzEventSchedulerTest#buildWithoutSchedulerThrowsAxonConfigurationException
[WARNING] org.axonframework.monitoring.MultiMessageMonitorTest#onMessageIngested_SingleMessageMonitor_success




________________________________
From: darko-marinov ***@***.***>
Sent: Thursday, October 17, 2024 7:34 PM
To: TestingResearchIllinois/idoft ***@***.***>
Cc: Trzaska, Weronika ***@***.***>; Author ***@***.***>
Subject: Re: [TestingResearchIllinois/idoft] 1 PR Accepted 1 PR Opened (PR #1342)


@darko-marinov commented on this pull request.

________________________________

In pr-data.csv<https://urldefense.com/v3/__https://github.com/TestingResearchIllinois/idoft/pull/1342*discussion_r1805648787__;Iw!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBFg7v6JN$>:

> @@ -3472,9 +3472,9 @@ https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810ae<https://urldefense.com/v3/__https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810ae__;!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBMUgyi8k$>
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_methodAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a<https://urldefense.com/v3/__https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_methodAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https:/*github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a__;Lw!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBC4ZwRvn$>
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a<https://urldefense.com/v3/__https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldCreateBeanFoo,OD,Deleted,,https:/*github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a__;Lw!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBIKVBHbN$>
 https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https://github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a<https://urldefense.com/v3/__https://github.com/awspring/spring-cloud-aws,178a524976521531272277b20a077b810aef19ff,spring-cloud-aws-context,org.springframework.cloud.aws.context.annotation.OnAwsCloudEnvironmentConditionTest.condition_typeAnnotation_shouldNotCreateBeanFoo,OD,Deleted,,https:/*github.com/awspring/spring-cloud-aws/commit/e7ac5a8121008a2e8ba22cbde2bb8c775493439a__;Lw!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBJ7X4UGF$>
-https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID<https://urldefense.com/v3/__https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID__;!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBMHnmKFD$>,,,
+https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,Opened,https://github.com/AxonFramework/AxonFramework/pull/3142<https://urldefense.com/v3/__https://github.com/AxonFramework/AxonFramework,e9809cbd5bb1f32b39d9e7408de1c8c395411350,messaging,org.axonframework.messaging.annotation.AnnotatedMessageHandlingMemberTest.attributeReturnsNonEmptyOptionalForMatchingAttributeKey,ID,Opened,https:/*github.com/AxonFramework/AxonFramework/pull/3142__;Lw!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBP2PqtFi$>,


3142 has been accepted. Did you run NonDex on the entire project to see if it finds some new tests?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/TestingResearchIllinois/idoft/pull/1342*pullrequestreview-2376643272__;Iw!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBMTrJDns$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/BFS5YYZLQFA2U54WDO2GBATZ4BJQPAVCNFSM6AAAAABQECCPM6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDGNZWGY2DGMRXGI__;!!DZ3fjg!5s5kNXhuWd8WlgcpBggAUeFTlzTLk8OOvD9avswgoBH8n76JLJDr6b7IiyoMt_b_ec38CAeSgV6xIPZ63-BdBGSmR7wh$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
